src/dmd/cppmanglewin.d

```diff
@@ -684,22 +684,148 @@ private:
             buf.writestring("?1");
             return;
         }
+        else if (sym.ident)
+        {
+            if (sym.ident.equals(Id.assign))
+            {
+                buf.writestring("?4");
+                return;
+            }
+            else if (sym.ident.equals(Id.eq))
+            {
+                buf.writestring("?8");
+                return;
+            }
+            else if (sym.ident.equals(Id.index))
+            {
+                buf.writestring("?A");
+                return;
+            }
+            else if (sym.ident.equals(Id.call))
+            {
+                buf.writestring("?R");
+                return;
+            }
+        }
         if (TemplateInstance ti = sym.isTemplateInstance())
         {
+            auto id = ti.tempdecl.ident;
+            const(char)* symName = id.toChars();
+
+            int firstTemplateArg = 0;
+
+            // test for special symbols
+            import std.algorithm : countUntil;
+            auto operators = [Id._cast, Id.assign, Id.eq, Id.index, Id.call, Id.opUnary, Id.opBinary, Id.opOpAssign];
+            sizediff_t whichOp = operators.countUntil!((o, n) => n.equals(o))(ti.name);
```

Comment:
@WalterBright does this make you upset? :)
Without this, I return to `if`/`else if`, and the thing gets a lot longer and uglier.

Reply:
We don't use phobos in the dmd source, mainly to try to keep dependencies down when dmd is ported to a new machine.